agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -99,7 +99,7 @@ if $check; then
 		:
 	else
 		for item in ${sysinfo//,/ }; do
-			if echo "${sysinfo_opts[@]}" | grep -q -w "$item"; then
+			if echo "${sysinfo_opts_default[@]}" | grep -q -w "$item"; then
```

Comment:
Why did you change this? Shouldn't you be checking the actual sysinfo opts, not the default ones?

Reply:
sysinfo has the actual passed sysinfo opts, we are checking if the passed opts are valid or not.